.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.4.1
```

Comment:
@franklinwise Interesting point, I didn't know it can try multiple Go versions. Still, I think in the future it'll be more important for us to compile against the vendored dependencies... so it looks good as it is now. Let's see if it works :)